src/main/java/com/ninecookies/wiremock/extensions/HttpCallbackHandler.java

```diff
@@ -61,34 +112,77 @@ private HttpCallbackHandler(ScheduledExecutorService executor, File callbackFile
     @Override
     public void handle(HttpCallbackDefinition callback) throws CallbackException {
         getLog().debug("CallbackHandler.run()");
+
+        URI uri = createURI(callback.target);
+        HttpContext context = createHttpContext(uri, callback.authentication);
+        HttpPost post = createPostRequest(uri, (String) callback.data);
```

Comment:
just for my education: PUT is not possible right now?

Reply:
Nope, not supported.. 
We would have to expose `method` on callback extension API, though this is planned to be done whenever necessary... ;)